spec/features/guide_spec.rb

```diff
@@ -60,18 +60,29 @@
   it "publishes guide editions" do
     fill_in_guide_form
 
-    expect(GdsApi::PublishingApiV2).to receive(:new).and_return(api_double).twice # save and update
+    expect(GdsApi::PublishingApiV2).to receive(:new).and_return(api_double).thrice
     expect(api_double).to receive(:put_content)
-                            .twice
+                            .thrice
                             .with(an_instance_of(String), be_valid_against_schema('service_manual_guide'))
     expect(api_double).to receive(:publish)
                             .twice
                             .with(an_instance_of(String), 'minor')
 
+    click_button "Save Draft"
+    guide = Guide.first
```

Comment:
Minor: this feels a bit brittle, I usually do something more specific to ensure the right record is returned, e.g. `Guide.find_by_slug`